packages/neuron-wallet/src/services/tx/transaction-generator.ts

```diff
@@ -477,6 +479,266 @@ public static generateSendingAllTx = async (
 
     return tx
   }
+
+  // sUDT
+  public static async generateCreateAnyoneCanPayTx(
+    tokenID: string,
+    lockHashes: string[],
+    blake160: string,
```

Comment:
These variable names (...blake160...) would be better to be more explicit. In this case, it would be more readable to indicate whether it is involving the sender or recipient.